stories/icon-message/icon-message.stories.js

```diff
@@ -0,0 +1,80 @@
+export default {
+    title: 'Components/Icon Message',
+    parameters: {
+        tags: ['f3', 'a11y', 'theme', 'development'],
+        description: `
+The icon-message component is usefull when using semantic icons or messages.
+##Usage
+**Use notifications if:**
+    
+* You want to show a semantic icon being error, success, warning, information, question.
+* You want to show a title using a semantic color
+            `,
+        components: ['icon-message']
+    }
+};
+
+export const success = () => `
+<div>
+<div class="fd-icon-message-success"></div>
+<div class="fd-icon-message-success fd-icon-message-success--no-icon">Success</div>
```

Comment:
Could you change it to
```suggestion
<i arial-label="success" class="fd-icon-message-success"></i>

<br/><br/>

<span class="fd-icon-message-success fd-icon-message-success--no-icon">Success</span>
```

Reply:
i fixed it with different naming convention but same items